src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -823,15 +823,33 @@ internal async Task<Collection<FunctionDescriptor>> GetFunctionDescriptorsAsync(
 
                 Utility.VerifyFunctionsMatchSpecifiedLanguage(functions, workerRuntime, _environment.IsPlaceholderModeEnabled(), _isHttpWorker, cancellationToken, throwOnMismatch: throwOnWorkerRuntimeAndPayloadMetadataMismatch);
 
+                var inProcIndexingSupported = _environment.IsPlaceholderModeEnabled()
+                                              || (_environment.IsDotNetInProcSupported()
+                                                  && !_hostingConfigOptions.Value.IsDotNetInProcDisabled);
+
                 foreach (FunctionMetadata metadata in functions)
                 {
                     try
                     {
-                        bool created = false;
+                        // If this is metadata represents a function that supports direct type indexing (in-proc), log a warning and skip it.
```

Comment:
If I'm reading the below code correctly, it looks like we won't skip the function like you say with a warning, but you throw an uncaught error?

Reply:
That's correct. The comment is stale from an initial version that was going to emit a warning. Will update.